commands/pack.go

```diff
@@ -56,24 +56,33 @@ func init() {
 }
 
 func packRun(flags packFlags) error {
-	buildpackDir, err := os.MkdirTemp("", "dup-dest")
+	buildpackOrExtensionDir, err := os.MkdirTemp("", "dup-dest")
 	if err != nil {
 		return fmt.Errorf("unable to create temporary directory: %s", err)
 	}
-	defer os.RemoveAll(buildpackDir)
+	defer os.RemoveAll(buildpackOrExtensionDir)
+
+	var buildpackOrExtensionTOMLPath string
```

Comment:
Similar to above, can we name this something shorter like `configPath`?

Reply:
sure :)